frontend/src/store/modules/user.ts

```diff
@@ -18,10 +18,14 @@ export default {
     setCurrentUser: (state: any, user: any) => {
       state.currentUser = user;
       if (!user) {
-        router.push({
-          path: "/login",
-          query: { redirect: router.currentRoute.fullPath }
-        });
+        router
+          .push({
+            path: "/login",
+            query: { redirect: router.currentRoute.fullPath }
+          })
+          .catch(() => {
+            // prevent uncaught promise error
+          });
```

Comment:
From the perspective of the task, this change looks irrelevant. Is it for fixing http://prntscr.com/prg5qm?

Reply:
Yes